zim/plugins/pageindex/__init__.py

```diff
@@ -126,6 +127,21 @@ def reload_model(self):
 		model = PageTreeStore(self.pageview.notebook.index)
 		self.treeview.set_model(model)
 
+	@action(_('Collapse All'), accelerator='<Alt>K', menuhints='view') # T: menu item
```

Comment:
Be aware that if the label is shown in the generic "view" menu, it won't be clear that "collapse all" relates to the index.

More general, afraid this won't scale - can't have a window-wide accelerator like this for each side pane. So probably should be a keybinding that is activated only when the index pane has focus.

Reply:
I see two options:
1) one option would be to remove it completely from "View" Menu (and only keep it as a shortcut)
2) another option would be to simply rename it ("Collapse Page Index") - I just changed that in the latest commit

I would really vote for an option that works independently from the current focus - as this should, from my point of view, work while one is typing in the ZIM notebook/editor as one does not need to change focus first to Pageindex.

BR

Reply:
PS: do you agree with the other feature that is part of the pull request:
"Added Mnemonics to unfold/select current page in PageIndex (useful e.g. when not using the autoexpand feature of the PageIndex Plugin)" ?
Thank you for your feedback & best regards